### PR TITLE
update(fsx_ext4_stress_test): verify for degraded state of faulted replica

### PR DESCRIPTION
**Pull request by rohan2794**
In case of restful control plane , when a replica is faulted via gRPC corresponding new replica created on the same pool with Degraded state before transitioning to Online state. So add degraded replica state check to wait for single Faulted or Degraded replica state.
JIRA: https://mayadata.atlassian.net/browse/MQ-2597

### CODE REVIEW

File: src/MQ-1783-fsx_ext4_stress/util.go [hunk 1]
```diff
@@ -280,7 +280,7 @@ func (c *fsxExt4StressConfig) waitForFsxPodCompletion() {
 
 // verify faulted replica
```
Comment (chriswldenyer) [inline src/MQ-1783-fsx_ext4_stress/util.go:281]: Maybe expand the comment to state that it waits for exactly one faulted or degraded replica.

**Review by chriswldenyer — APPROVED** [review]
LGTM, one suggestion

**Review by blaisedias — APPROVED** [review]
LGTM.
Can you add the description from the JIRA to the git commit and PR please.

File: src/MQ-1783-fsx_ext4_stress/util.go (hunk 1 above)
Comment (rohan2794) [reply src/MQ-1783-fsx_ext4_stress/util.go]: Done

**Pull request merged**